src/Roave/SecurityAdvisories/Version.php

```diff
@@ -40,17 +41,17 @@ final class Version
      */
     private function __construct(array $matches)
     {
-        $this->versionNumbers = self::removeTrailingZeroes(...array_map('intval', explode('.', $matches['version'])));
+        $this->versionNumbers = self::removeTrailingZeroes(...array_map('\intval', explode('.', $matches['version'])));
```

Comment:
This is wrong: strings are already references to the FQN. The tool has a false positive here

Reply:
Wow, this is confusing! Good to know though. I'll revert that and create an issue for that FP

Reply:
reverted and reported: https://github.com/kalessil/phpinspectionsea/issues/1524